### PR TITLE
ai course update

### PR DESCRIPTION
**Pull request by Morbiusss**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Review by LuigiImVector — COMMENTED** [review]
sounds like English to me

File: courses/free-courses-hi.md [hunk 1]
```diff
@@ -85,6 +85,8 @@
 
 * [Artificial Intelligence (Complete Playlist)](https://www.youtube.com/playlist?list=PLxCzCOWd7aiHGhOHV-nwb0HR5US5GFKFI) - Gate Smashers
 * [Artificial Intelligence Lectures Hindi](https://www.youtube.com/playlist?list=PLV8vIYTIdSnYsdt0Dh9KkD9WFEi7nVgbe) - Easy Engineering Classes
+* [Playlist to Artificial Intelligence 🔥🔥](https://www.youtube.com/playlist?list=PLPIwNooIb9vgB1DQEftkKA3qOdeC4vonA) -
```
Comment (eshellman) [inline courses/free-courses-hi.md:88]: remove the emojis. 
Please read docs/CONTRIBUTING.md guidelines!

**Comment by punyakrit** [comment]
The changes in this pull request has already been made.

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**